src/pages/variantEntity/TabSummary.tsx

```diff
@@ -1,49 +1,216 @@
-import React, { FC } from 'react';
-import Card from '@ferlab/ui/core/view/GridCard';
+import React from 'react';
 import { useTabSummaryData } from 'store/graphql/variants/tabActions';
-import { Spin } from 'antd';
+import { List, Result, Space, Spin, Table } from 'antd';
+import Summary from './Summary';
+import StackLayout from '@ferlab/ui/core/layout/StackLayout';
+import { Consequence, VariantEntity } from 'store/graphql/variants/models';
 
 type OwnProps = {
   variantId: string;
 };
 
-const TabSummary: FC<OwnProps> = ({ variantId }) => {
-  const { loading } = useTabSummaryData(variantId);
+type TableGroup = {
+  consequences: Consequence[];
+  omim: string;
+  biotype: string;
+  symbol: string;
+};
+
+type SymbolToConsequences = {
+  [key: string]: TableGroup;
+};
+
+const groupConsequencesBySymbol = (consequences: Consequence[]) => {
+  if (consequences.length === 0) {
+    return {};
+  }
+  return consequences.reduce((acc: SymbolToConsequences, consequence: Consequence) => {
+    const symbol = consequence.node.symbol;
+    if (!symbol) {
+      return acc;
+    }
+    const omim = consequence.node.omim_gene_id || '';
+    const biotype = consequence.node.biotype || '';
+    const oldConsequences = acc[symbol]?.consequences || [];
+
+    return {
+      ...acc,
+      [symbol]: {
+        consequences: [...oldConsequences, { ...consequence }],
+        omim,
+        biotype,
+        symbol,
+      },
+    };
+  }, {});
+};
+
+const orderGenes = (mSymbolToConsequences: SymbolToConsequences) => {
+  if (!mSymbolToConsequences || Object.keys(mSymbolToConsequences).length === 0) {
+    return [];
+  }
+  return Object.entries(mSymbolToConsequences).map(([, values]) => {
+    return { ...values };
+  });
+};
+
+const orderConsequences = (consequences: Consequence[]) => [...consequences];
```

Comment:
rules are not clear yet from spec. However, consequences will have to be ordered so I left a placeholder